docs/loader_usage.md

```diff
@@ -0,0 +1,114 @@
+- [scylla-bench](https://github.com/scylladb/scylla-bench/) - a newer stress tool designed for Scylla, providing more advanced features especially for testing large partitions.
+- [gemini](https://github.com/scylladb/gemini/) - tool for data consistency validation across various schemas and mixed workloads (read, write, delete). Verifies data integrity against 'oracle' cluster.
+- [latte](https://github.com/scylladb/latte/) - advanced stress tool, mostly used for custom schemas (supports many data types and is very flexible). `rust` based.
+- [cql-stress](https://github.com/scylladb/cql-stress/) - `rust` based equivalent of `cassandra-stress` and `scylla-bench` - supports the same command line options.
+- [ycsb](https://github.com/scylladb/ycsb/) - only tool supporting ScyllaDB Alternator API, used for testing ScyllaDB in AWS DynamoDB compatible mode.
+
+Loaders execute these stress tools within Docker containers (Docker images specified in `defaults/docker_images`). They help simulate realistic database loads and measure performance and reliability.
+
+---
+
+## How Loaders Operate
+
+The stress load is initiated usually via methods such as:
+
+- `sdcm.tester.ClusterTester.run_stress_thread`
+- `sdcm.utils.loader_utils.LoaderUtilsMixin.assemble_and_run_all_stress_cmd`
+
+### Stress Command Parameters
+
+Stress commands are specified using these parameters:
+
+- **`stress_cmd`**: Used primarily in longevity tests.
+- **`stress_cmd_<w|r|m>`**: Used in performance tests, with `<w|r|m>` representing write, read, or mixed workloads.
+
+These parameters accept either a single string or a list of strings, each representing a distinct stress command.
+
+### Adjustments to Stress Commands
+
+Stress commands provided by the user undergo the following modifications before execution:
+
+- **Node targeting**: IP addresses of all nodes in the cluster are appended automatically.
+- **Duration adjustment**: Modified according to the test’s `stress_duration` and `prepare_stress_duration` parameters, if present.
+- **HDR Histogram**: if `use_hdrhistogram` is set to `true`, proper arguments are added and SCT handles collection HDR histograms. Supported by `cassandra-stress`, `cql-stress`, `latte` and (possibly) `scylla-bench`; (`ycsb` WIP).
+- **Authentication**: Adjusted based on LDAP parameters (`use_ldap`, `are_ldap_users_on_scylla`) and `authenticator_user` and `authenticator_password`.
+- **Encryption**: Included when `client_encrypt` is enabled, considering additional params like `peer_verification` and `client_encrypt_mtls`.
+- **Multi-DC tests**: Loader targets nodes within the same datacenter.
+- **Rack awareness**: Enabled if `rack_aware_loader` is `true` (stress command targets nodes on the same rack as loader is).
+
+### Execution Modes
+
+Commands are executed in two possible modes based on the `round_robin` setting:
+
+- **Round Robin (`true`)**: Each stress command runs sequentially on loaders, rotating through available loaders.
```

Comment:
If I didn't know the behavior of the `round_robin=true` I would understand this statement in a wrong way.
`sequentially` may be understood as `execution_pool=1`, which is false.

Can we rephrase it to something like following:
```
Round Robin (true): Each stress command runs once on a loader picked up using the `round_robin` selection approach.
```